### ISSUE TITLE
In VS-Code when clicking on the error location it is not parsed correctly

### ISSUE THREAD

**Issue by tronta** [hashintel/hash#1173]
### Describe the bug

In your example
```
Error: experiment error: could not run experiment
├╴src/main.rs:74:18
├╴unable to set up experiments
│
├─▶ invalid experiment description
│   ├╴src/main.rs:44:10
│   ╰╴experiment 2 could not be parsed
```
I clicked on `src/main.rs:44:10` but the IDE has problems in interpreting it and prompts that it does not find the location:
![grafik](https://user-images.githubusercontent.com/5135577/194400100-e08b340b-94bb-435e-9cf5-7f4e88e3ebe5.png)

Probably it's due to the fact, that there is no space before `src` like `├╴ src/main.rs:44:10`

### To reproduce

Click on location `├╴src/main.rs:44:10` in error report.

### Expected behavior

Editor jumps to the correct position in the source code.
Probably no so compact nice, but putting a space behind `├╴` or `╰╴` would help.

**Comment by indietyp** [comment]
Thank you very much for your issue! 

After some investigation into what caused the issue, we have issued a fix in #1174, which should resolve the issue in the next version of error-stack.

**Comment by TimDiekmann** [comment]
This is now available in error-stack v0.2.2
